src/app/components/DigitalBill/Components/DigitalBillModal/index.js

```diff
@@ -0,0 +1,141 @@
+import React, { Fragment, useState } from 'react';
+import i18 from 'i18next';
+import { UTLabel, UTButton, UTTextInput, UTLoading } from '@widergy/energy-ui';
+import { connect } from 'react-redux';
+import { bool, string } from 'prop-types';
+
+import DigitalBillsActions from 'redux/bills/actions';
+import { checkEmail } from 'utils/checkEmailsUtils';
+import { digitalTypes } from 'types/digitalTypes';
+
+import styles from './styles.module.scss';
+
+const DigitalBillModal = ({ show, type, email, onHide, digitalBills, loading, error, dispatch }) => {
+  const [modifyInput, setModifyInput] = useState('');
+  const [emailIsValid, setEmailIsValid] = useState(false);
+
+  const handleModifyInput = inputEmail => {
+    setEmailIsValid(checkEmail(inputEmail));
+    setModifyInput(inputEmail);
+  };
+
+  const handleConfirm = typeAction => {
+    switch (typeAction) {
+      case 'modificar':
+        dispatch(DigitalBillsActions.putDigitalBills(modifyInput));
+        break;
+
+      case 'baja':
```

Comment:
extraé "baja", "alta" y "modificar" a un archivo de constants dentro de la carpeta constants.js en el root del proyecto y utilizalas en todos los casos donde la necesites.

Te debería quedar algo así:

export const DELETE_METHOD: 'baja';

y entonces en este caso deberías importarte esa constante acá y usarla así: 
```suggestion
      case : DELETE_METHOD:
```

Reply:
Perfecto, realizado!